trafficFlowTests.py

```diff
@@ -109,13 +109,13 @@ def _run_tests(
         for tasks in servers + clients + monitors:
             tasks.setup()
 
-        for tasks in servers + clients + monitors:
+        for tasks in clients + monitors:
             tasks.run(duration)
```

Comment:
This change seems wrong. Tasks should be treated all the same, and if a certain task's `run()` has nothing to do, it should just do nothing. But the callers shouldn't know that, they would still call `run()`.

Granted, you also replaced `pass` in
```python
    def run(self, duration: int) -> None:
        raise NotImplementedError("run() not implemented in IperfServer")
```
But why? If there is nothing to do for that type, then it should just silently do nothing and not raise an exception.

Reply:
That makes sense. I thought we wouldn't want to call empty functions, but it does break the pattern. I'll change it back.